<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarginalNotesDetectionClass</class>
 <widget class="QMainWindow" name="MarginalNotesDetectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarginalNotesDetection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsViewImage"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonGrabCut">
          <property name="text">
           <string>GrabCut</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonChooseImage">
          <property name="text">
           <string>Choose Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxInteractive">
          <property name="text">
           <string>Interactive</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditImageName"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="marginalnotesdetection.qrc"/>
 </resources>
 <connections/>
</ui>
